.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+- repo: https://github.com/asottile/add-trailing-comma
+  rev: v0.6.4
+  hooks:
+  - id: add-trailing-comma
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v1.3.0
+  hooks:
+  - id: trailing-whitespace
+    args: ['--markdown-linebreak-ext=md,rst']
```

Comment:
interesting! you may be the first I've seen using this setting 😆

Reply:
TBH, I wasn't sure I needed this, but figured I'd add it since we're using rST on this project. Might be worth supporting `rst` by default, since many Python projects use it.

Reply:
the commandline option is for an _obscure_ (imo anti-feature) of markdown in that you can force a linebreak by *intentionally* adding trailing whitespace

I've yet to find actual code in the wild which uses it (I checked out your branch and removed this and it doesn't change anything for `marshmallow` either)

Reply:
Ah ok, I'll just remove it then. Thanks.